packages/dinero.js/etc/dinero.js.api.md

```diff
@@ -0,0 +1,173 @@
+## API Report File for "dinero.js"

```

Comment:
This file is generated by api-extractor. From what I understand it is generally used as a way to quickly highlight API changes for project maintainers. Making it quick to see if a change modifies the external api.